internal/gps/pkgtree/pkgtree.go

```diff
@@ -64,17 +65,15 @@ func ListPackages(fileRoot, importRoot string) (PackageTree, error) {
 		ImportRoot: importRoot,
 		Packages:   make(map[string]PackageOrErr),
 	}
+	mu := &sync.Mutex{}
```

Comment:
sure, is there a style guide for this project? i know many people have different opinions. i'll change this one